scripts/notify.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+## usage:
+# ./notify.py FAILURE
+# ./notify.py SUCCESS
+
+import os
+import json
+import sys
+import requests
+import datetime
+from logsnag import LogSnag
+import slack_notifications as slack
+
+
+#the 4 notification channels:
+# - slack
+# - notifi (works for both phone and mac)
+# - push on the phone (via 'logsnag', pushme disappeared)
```

Comment:
```suggestion
# - push on the phone (via 'logsnag')
```

Might be good to mention it in the PR description, but I don't think it's necessary here